backend/editor/entries.py

```diff
@@ -241,7 +241,20 @@ def close_project(self):
         params = {"project_name": self.project_name, "status": "CLOSED"}
         get_current_transaction().run(query, params)
 
-    def list_existing_projects(self):
+    def reopen_project(self):

```

Comment:
there again, do we want a reopen_project method, or just a set_status method ?

If there are a lot of pre/post processing specific to that case, it make sense to have a specific method (for example, if we were sending an email, etc.) but if not ain't it better to have a generic "set_status" method ?

Reply:
Yep, it should be better. So, I changed it to a set_status() method and used it for closing a project as well.